.pipelines/build-pipelines.yml

```diff
@@ -38,15 +38,25 @@ steps:
 # We cannot set this in variables section above because $(isNugetRelease) 
 # is not available at pipeline compilation time.
 - bash: |
-    echo ISNUGETRELEASE = $ISNUGETRELEASE
+    echo IsNugetRelease = $ISNUGETRELEASE
+    echo IsReleaseCandidate = $ISRELEASECANDIDATE
+    echo RELEASE_CANDIDATE_NUMBER = $RELEASE_CANDIDATE_NUMBER
```

Comment:
Any way we can autoassign RELEASE_CANDIDATE_NUMBER?

Reply:
As of now, I couldn't find any direct method to do so. Although there are other complex approaches that requires parsing etc. I'll create an issue and implement it if I find a better approach.